sormas-backend/src/test/java/de/symeda/sormas/backend/travelentry/TravelEntryFacadeEjbPseudonymizationTest.java

```diff
@@ -0,0 +1,163 @@
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.emptyString;
+import static org.hamcrest.Matchers.is;
+
+import java.util.HashMap;
+
+import javax.persistence.Query;
+
+import org.junit.jupiter.api.Test;
+
+import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.caze.CaseDataDto;
+import de.symeda.sormas.api.feature.FeatureConfigurationIndexDto;
+import de.symeda.sormas.api.feature.FeatureType;
+import de.symeda.sormas.api.feature.FeatureTypeProperty;
+import de.symeda.sormas.api.person.PersonDto;
+import de.symeda.sormas.api.travelentry.TravelEntryDto;
+import de.symeda.sormas.api.user.DefaultUserRole;
+import de.symeda.sormas.api.user.UserDto;
+import de.symeda.sormas.api.utils.DataHelper;
+import de.symeda.sormas.backend.AbstractBeanTest;
+import de.symeda.sormas.backend.TestDataCreator;
+import de.symeda.sormas.backend.feature.FeatureConfiguration;
+
+public class TravelEntryFacadeEjbPseudonymizationTest extends AbstractBeanTest {
+
+	private TestDataCreator.RDCF rdcf1;
+	private TestDataCreator.RDCF rdcf2;
+
+	private UserDto nationalUser;
+
+	@Override
+	public void init() {
+		super.init();
+		rdcf1 = creator.createRDCF("Region 1", "District 1", "Community 1", "Facility 1", "Point of entry 1");
+		rdcf2 = creator.createRDCF("Region 2", "District 2", "Community 2", "Facility 2", "Point of entry 2");
+		nationalUser = creator.createUser(
+			rdcf1.region.getUuid(),
+			rdcf1.district.getUuid(),
+			rdcf1.community.getUuid(),
+			rdcf1.facility.getUuid(),
+			"Nat",
+			"User",
+			creator.getUserRoleReference(DefaultUserRole.NATIONAL_USER));
+	}
+
+	@Test
+	public void testPseudonymizedGetByUuidWithLimitedUser() {
+
+		// deactivate AUTOMATIC_RESPONSIBILITY_ASSIGNMENT in order to assign the limited user to a case from outside jurisdiction
+		FeatureConfigurationIndexDto featureConfiguration =
+			new FeatureConfigurationIndexDto(DataHelper.createUuid(), null, null, null, null, null, true, null);
+		getFeatureConfigurationFacade().saveFeatureConfiguration(featureConfiguration, FeatureType.CASE_SURVEILANCE);
+
+		executeInTransaction(em -> {
+			Query query = em.createQuery("select f from featureconfiguration f");
+			FeatureConfiguration singleResult = (FeatureConfiguration) query.getSingleResult();
+			HashMap<FeatureTypeProperty, Object> properties = new HashMap<>();
+			properties.put(FeatureTypeProperty.AUTOMATIC_RESPONSIBILITY_ASSIGNMENT, false);
+			singleResult.setProperties(properties);
+			em.persist(singleResult);
+		});
+
+		loginWith(nationalUser);
+
+		// travel entry within limited user's jurisdiction
+		PersonDto person1 = creator.createPerson("John", "Doe");
+		TravelEntryDto travelEntry1 = creator.createTravelEntry(
+			person1.toReference(),
+			nationalUser.toReference(),
+			Disease.CORONAVIRUS,
+			rdcf1.region,
+			rdcf1.district,
+			rdcf1.pointOfEntry);
+		travelEntry1.setQuarantineHomePossibleComment("pacient can stay home");
```

Comment:
please use the creator method with a consumer so you don't need to update the travel entry right after creation

Reply:
changed the creator method to the one that has a consumer